test/DADBridgeWrapper.ts

```diff
@@ -109,55 +103,48 @@ describe('DADBridgeWrapper', () => {
             await expect(dadBridgeWrapper.connect(deployer).mint(user.address, MINT_AMOUNT)).to.be.revertedWith(
                 `AccessControl: account ${deployer.address.toLowerCase()} is missing role ${ROLE_ADMIN}`
             );
-            await assertState(TOTAL_SUPPLY, BigNumber.from(0));
         });
 
         it('admin should not be able to mint to an invalid recipient', async () => {
             await assertState(TOTAL_SUPPLY, BigNumber.from(0));
             await expect(dadBridgeWrapper.connect(admin).mint(INVALID_RECIPIENT, MINT_AMOUNT)).to.be.revertedWith(
-                `InvalidRecipient("${INVALID_RECIPIENT}")`
+                `InvalidRecipient()`
```

Comment:
We generally don't add parentheses here, but I won't die on this hill

Reply:
Fixed, but FYI - when we do start adding parameters, those empty brackets will fail the test in the case of an error with parameters, thus letting you know that you can improve the test by adding the expected parameter values to it (whereas, without those brackets, the test will pass because the error name will consist a sufficient prefix). Also, adding the `()` will be a more accurate coding convention once we have errors with parameters.

Reply:
Agree with everything that you've said and had similar thoughts in the past, but since it's doesn't seem that likely for us to use custom errors with parameters (e.g., we didn't have a single good use-case for it in whole V3) - let's avoid adding extra `()` for now